src/streams.c

```diff
@@ -736,6 +736,10 @@ static Obj PRINT_OR_APPEND_TO(Obj args, int append)
     i = append ? OpenAppend( CSTR_STRING(filename) )
                : OpenOutput( CSTR_STRING(filename) );
     if ( ! i ) {
+        if (strcmp(CSTR_STRING(filename), "*errout*")) {
+            // When trying to print an error opening *errout* failed
+            SyExit(1);
```

Comment:
Good on, I actually run into this problem recently. An error message before exiting would be helpful, though, e.g.
```C
  fputs("gap: panic, could not open *errout*!\n", stderr);
  SyExit(1);
```

Reply:
I will add that. I don't think it will actually output anything (as errout is usually stderr, so if we can't print to errout, we probably can't print to stderr either). However, it doesn't hurt to try!